docker/intel/Dockerfile-18

```diff
@@ -30,33 +30,25 @@ COPY intel.cfg /tmp/
 RUN cd /tmp \
  && mkdir intel \
  && cd intel \
- && curl -sL http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/11541/parallel_studio_xe_2017_update4_composer_edition.tgz | tar --strip-components=1 -xz \
+ && curl -sL http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/13717/parallel_studio_xe_2018_update4_cluster_edition_online.tgz | tar --strip-components=1 -xz \
  && USER=root ./install.sh --silent /tmp/intel.cfg \
  && cd \
- && rm -r /tmp/intel
+ && rm -r /tmp/intel \
+ && rm -r /opt/intel/inspector* /opt/intel/vtune_amplifier* /opt/intel/ism /opt/intel/clck /opt/intel/advisor /opt/intel/documentation* \
+ /opt/intel/debugger* /opt/intel/itac /opt/intel/compilers_and_libraries_2018.5.274/linux/daal /opt/intel/compilers_and_libraries_2018.5.274/linux/ipp /tmp/parallel_studio_xe_2018_update4_cluster_edition
```

Comment:
This is ridiculous. Just properly select the components you install.

Reply:
If you tell me the names for intel18, I don't have time to look them up.